README.md

```diff
@@ -0,0 +1,198 @@
+<a name="readme-top"></a>
+
```

Comment:
- [ ] OPTIONAL: In the mobile view, the new hamburger menu is overflowing the "My Appointments" header:

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/132f0a8f-6650-48c5-983d-adf42513ea18)

Kindly, make sure the content is rendered a little lower on the screen so the hamburger menu is free.